_includes/sidebar.html

```diff
@@ -81,6 +81,7 @@
 
       {% if url %}
       <a href="{{ url }}" aria-label="{{ entry.type }}"
+        {%- if entry.type == 'mastodon' %} rel="me"{% endif %}
```

Comment:
If two `rel` attributes exist at the same time, their values should be placed in a list. For example: `rel="val_a val_b"`

Reply:
Good point, updated. Any thoughts on a better implementation though?

Reply:
I think so, how about this?

```diff
       {% if url %}
+        {% assign link_types = nil %}
       <a href="{{ url }}" aria-label="{{ entry.type }}"
-        {% unless entry.noblank %}target="_blank" rel="noopener"{% endunless %}>
+        {% unless entry.noblank %}
+          {% assign link_types = link_types | append: " noopener" %}
+          target="_blank"
+        {% endunless %}
+
+        {% if entry.type == 'mastodon' %}
+          {% assign link_types = link_types | append: " me" %}
+        {% endif %}
+
+        {% if link_types %}rel="{{ link_types | lstrip }}"{% endif %}>
         <i class="{{ entry.icon }}"></i>
       </a>
       {% endif %}
```

Reply:
Looks great! Updated and added you as a co-author.